lib/galaxy/tool_util/verify/wait.py

```diff
@@ -20,7 +22,7 @@ def wait_on(
     timeout: timeout_type,
     delta: timeout_type = DEFAULT_POLLING_DELTA,
     polling_backoff: timeout_type = DEFAULT_POLLING_BACKOFF,
-    sleep_: Optional[Callable] = None,
+    _sleep: Optional[Callable] = None,  # only used in tests
```

Comment:
```suggestion
    sleep_: Optional[Callable] = None,  # only used in tests
```
This being a public function in a published package I wouldn't change the name

Reply:
done